pegasus/sites.v3/code.org/views/about_headshots.haml

```diff
@@ -5,9 +5,9 @@
       - avatar_url = avatar_image(person[:name_s], 200, true)
       .col-33.avatar_container{:id => person[:id]}
         - unless person[:url_s].nil_or_empty?
-          %a{:href=>person[:url_s], :target=>"_self"}
+          %a{:href=>person[:url_s], :target=>"_blank"}
```

Comment:
I assume we want personal websites/extended bios opening in new windows? Open to feedback here!

Reply:
There are some security concerns about using target blanks, but you can use `rel: "noopener noreferrer"` here to mitigate that.

Reply:
there are also accessibility issues with using target=_blank. [this article](https://dev.to/cerchie/3-ways-to-make-target-blank-accessible-3lan) mentions some workarounds

Reply:
Thank you both! Alternate idea: I could leave it as `:target=>"_self"` since _most_ of these links are internal anyway?

Reply:
That works for me! Especially since these links will probably be deprecated soon 😅